<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <widget class="QLineEdit" name="states_lineedit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="states_label">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>states</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="symbols_lineedit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="symbols_label">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>90</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>symbols</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="start_state_lineedit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="start_state_label">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>start state</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="end_states_lineedit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="end_states_label">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>210</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>end states</string>
    </property>
   </widget>
   <widget class="QLabel" name="transitions_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>transitions</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="transitions_textedit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>231</width>
      <height>321</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="dfa_output_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>dfa output</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="dfa_output_textedit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>421</width>
      <height>371</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="error_lineedit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>460</y>
      <width>321</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="error_label">
    <property name="geometry">
     <rect>
      <x>377</x>
      <y>470</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>error</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_button">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>550</y>
      <width>101</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="step_button">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>550</y>
      <width>101</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>step</string>
    </property>
   </widget>
   <widget class="QPushButton" name="process_button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>550</y>
      <width>101</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>process</string>
    </property>
   </widget>
   <widget class="QLabel" name="dfa_state_label">
    <property name="geometry">
     <rect>
      <x>377</x>
      <y>510</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>dfa state</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="dfa_state_lineedit">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>500</y>
      <width>321</width>
      <height>32</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>-10</y>
      <width>20</width>
      <height>621</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="input_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>input</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="input_lineedit">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>381</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
